runtime/src/main/java/org/corfudb/protocols/wireprotocol/CorfuMsgType.java

```diff
@@ -88,6 +90,8 @@
     MANAGEMENT_FAILURE_DETECTED(74, new TypeToken<CorfuPayloadMsg<FailureDetectorMsg>>(){}, true),
     HEARTBEAT_REQUEST(75, TypeToken.of(CorfuMsg.class), true),
     HEARTBEAT_RESPONSE(76, new TypeToken<CorfuPayloadMsg<byte[]>>(){}, true),
+    ORCHESTRATOR_REQUEST(77, new TypeToken<CorfuPayloadMsg<AddNodeRequest>>() {}),
```

Comment:
Add javadoc for each message type

Reply:
Also, shouldn't the payload be a more generic type. AddNodeRequest seems limiting.

Reply:
Our rpc layer doesn't handle polymorphism nicely. For example, cant do something like
new TypeToken<CorfuPayloadMsg<OrchestratorRequest>> .

Reply:
First of all, TypeToken<CorfuPayloadMsg> doesn't tell it the message type.
Second of all, just define NodeRequest, put a type field in it, and a RequestData object in it.

Reply:
I don't think I completely understand your point. Are you saying that I should create one "container" message for the orchestrator and have a "payload" field in it that can be deserialized into different orchestrater requests ?

Reply:
Sure, that sounds about right.

Reply:
Done.